README.md

```diff
@@ -2,3 +2,136 @@
 
 This document explains our approach to working with Rubocop in dxw. We should
 in general:
```

Comment:
Shall we link to Rubocop at the top? https://github.com/bbatsov/rubocop/

Reply:
Fixed inline